webhook_to_fedora_messaging/endpoints/util.py

```diff
@@ -0,0 +1,60 @@
+from flask import Response, Request
+from ..config.defaults import FASJSON_URL
+import fasjson_client
+
+
+def not_found() -> Response:
+    return Response({'message': 'Not Found'}, status=404, mimetype='application/json')
```

Comment:
This is equivalent to doing `return {'message', 'Not Found'}, 404`:
- if you return a dict from a view, Flask will automatically convert it to JSON and set the Content-Type header
- if you return a tuple from a view with a number as the second element, Flask will use it as the return code.

Please look at [this Flask doc about error handling in API servers](https://flask.palletsprojects.com/en/2.3.x/errorhandling/#returning-api-errors-as-json)

Reply:
I'm going to fix it here https://github.com/fedora-infra/webhook-to-fedora-messaging/pull/32/files